examples/double-conversion/main.cpp

```
@@ -0,0 +1,6 @@
+#include <double-conversion/double-conversion.h>
```

Looks like `*Config.cmake` is not correct:
```
fatal error: double-conversion/double-conversion.h: No such file or directory
```

---

Not sure what you mean. It works for me using this command:

`TOOLCHAIN=default PROJECT_DIR=examples/double-conversion ./jenkins.py`

---

> It works for me using this command

I can reproduce error on Travis:
* https://travis-ci.org/ingenue/hunter/builds/271825246?utm_source=github_status&utm_medium=notification

---

Then why does only one build fail [here](https://travis-ci.org/ruslo/hunter/builds/271757246)?

---

That build failure I referenced is actually a documentation error. Fixed in the change I just pushed.

---

> Then why does only one build fail here?

It's different testing - only unit-tests and documentation, `example/double-conversion` is not tested.

---

You are right. I had double-conversion installed locally, masking the error. Thanks. Will fix soon.